tests/unit/keyinput/test_basekeyparser.py

```diff
@@ -342,3 +344,13 @@ def test_respect_config_when_matching_counts(keyparser, config_stub):
 
     assert not keyparser._sequence
     assert not keyparser._count
+
+
+def test_count_limit_exceeded(handle_text, keyparser, caplog):
```

Comment:
Does it? I tried commenting out your code above and it seems to fail without any excessive logging.

Reply:
`caplog.at_level(logging.ERROR)` might swallow the spam (I introduced that later).
IIRC these were debug messages from `BaseKeyParser`.

Reply:
Oh, now I remember. Actually in the beginning I was using `prompt_keyparser` in the test case, which handled the number keys individually instead of aggregating it as a count.